content/docs/start/data-and-model-versioning.md

```diff
@@ -82,35 +82,31 @@ outs:
     path: data.xml
 ```
 
-> \* See
-> [Large Dataset Optimization](/doc/user-guide/large-dataset-optimization) and
-> `dvc config cache` for more info. on file linking.
-
 </details>
 
 ## Storing and sharing
 
 You can upload DVC-tracked data or model files with `dvc push`, so they're
 safely stored [remotely](/doc/command-reference/remote). This also means they
 can be retrieved on other environments later with `dvc pull`. First, we need to
-setup a storage:
+setup a storage provider:
 
 ```dvc
 $ dvc remote add -d storage s3://mybucket/dvcstore
 $ git add .dvc/config
 $ git commit -m "Configure remote storage"
 ```
 
-> DVC supports the following remote storage types: Google Drive, Amazon S3,
-> Azure Blob Storage, Google Cloud Storage, Aliyun OSS, SSH, HDFS, and HTTP.
-> Please refer to `dvc remote add` for more details and examples.
+> DVC supports many remote storage types, including: Google Drive, Amazon S3,
+> Azure Blob Storage, Google Cloud Storage, Aliyun OSS, SSH, HDFS, and HTTP. See
```

Comment:
👍 but I'd either remove a few examples or keep the previous form and make it inclusive by adding the one main missing type (WebDAV).

Reply:
Would be hard to keep up-to-date. I prefer adding everything current whenever revised but keep the language leaving room for future additions. What would you suggest dropping from the examples? Aliyun? One of the Googles? HDFS?

Reply:
Remotes don't change that often and we have to maintain other full lists anyway. But fine.

I'd say keep S3, SSH, GDrive, maybe GCS, Azure, HDFS (in that order). I have the impression that those are the most use ones.